subt/ros/robot/launch/x4.launch

```diff
@@ -145,15 +145,9 @@
   <node pkg="nodelet" type="nodelet" name="pcl_manager" args="manager" output="screen" />
 
   <!-- Run a passthrough filter to remove drone propellers -->
-  <node pkg="nodelet" type="nodelet" name="passthrough" args="load pcl/PassThrough pcl_manager" output="screen">
-    <remap from="~input" to="/$(arg robot_name)/rgbd_camera/depth/points" />
-    <remap from="/passthrough/output" to="/$(arg robot_name)/rgbd_camera/depth/points_cleaned" />
-    <rosparam>
-      filter_field_name: z
-      filter_limit_min: 0.31
-      filter_limit_max: 100.0
-      filter_limit_negative: False
-    </rosparam>
+  <node name="filter_pointcloud" pkg="robot" type="filter_pointcloud.py">
+    <remap from="/A100LXM/rgbd_camera/depth/points" to="/$(arg robot_name)/rgbd_camera/depth/points"/>
```

Comment:
this long topic name was used for direct debugging without launch file. I am not sure if initial / should be there or not?

Reply:
This remains to be done.